src/main/java/cloud/fogbow/ras/core/plugins/interoperability/googlecloud/attachment/v1/GoogleCloudAttachmentPlugin.java

```diff
@@ -30,8 +30,11 @@
 public class GoogleCloudAttachmentPlugin implements AttachmentPlugin<GoogleCloudUser> {
     private static final Logger LOGGER = Logger.getLogger(GoogleCloudAttachmentPlugin.class);
 
-    @VisibleForTesting
-    static final String EMPTY_STRING = "";
+
+    //To delete an instance the body must be empty.
+    //Somehow content_length is not added to headers when the request body is empty.
+    //So the request will return a code > 204 (HttpStatus.SC_NO_CONTENT) and CloudHttpClient will throw an exception.
+    static final String SPECIAL_EMPTY_BODY = "{null:null}";
```

Comment:
Does the above comment still make sense? If not, remove. If yes, then perhaps you might want to rephrase it to make it clearer.